### DISCUSSION TITLE
ecs 附加 ECS 至应答的疑惑

### DISCUSSION THREAD

**Discussion by liang-hiwin** [IrineSistiana/mosdns#565] [Q&A]
v5 文档写着

```
ecs
附加 ECS 至应答。支持 IPv4/6，会自动附加在 A/AAAA 请求中。
参数:  ecs [ip/mask] [ip/mask]
示例:  ecs 1.2.3.4/24 2001:dd8:1a::1/48
```
请问如果自动附加客户端ip如何填写？
 这样子写？ ecs 0.0.0.0/0  ::0/0

**Comment by urlesistiana** [comment]
删了。因为 mosdns 不能有效的缓存带 ecs 的请求。作为服务器建议前置一个 unbound 。

**Reply by xukecheng** [reply]
请问意思是前置一个服务，然后发送请求给 Mosdns 并在请求中附带 ECS 吗？目前我有用 AdGuard Home，开启了 启用 EDNS 客户端子网，但是似乎还是没有生效 0.0
